### PR TITLE
Set logger and logger messages. Updated Injector (new version)

### PR DESCRIPTION
**Pull request by fyarkiy**

### CODE REVIEW

File: src/main/java/com/cinema/Main.java [hunk 1]
```diff
@@ -49,36 +51,41 @@ public static void main(String[] args) {
...
         tomorrowSession.setShowTime(LocalDateTime.of(2020, 10, 11, 21, 00, 00));
         movieSessionService.add(tomorrowSession);
+        logger.info("tomorrow session " + tomorrowSession + " was added");
 
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach((ms) -> logger.info(ms.toString()));
         cinemaHallService.getAll().forEach(System.out::println);
         movieSessionService.findAvailableSessions(1L, LocalDate.of(2020, 10, 10))
-                .forEach(System.out::println);
+                .forEach((ms) -> logger.info(ms.toString()));
```
Comment (vremere-a) [inline src/main/java/com/cinema/Main.java]: you can also use method - reference 
if you want☺️

**Review by vremere-a — APPROVED** [review]
good job👍

File: src/main/java/com/cinema/Main.java [hunk 2]
```diff
@@ -49,36 +51,42 @@ public static void main(String[] args) {
...
         afternoonSession.setShowTime(LocalDateTime.of(2020, 10, 10, 16, 00, 00));
         movieSessionService.add(afternoonSession);
+        logger.info("Afternoon session " + afternoonSession + " was added");
 
         MovieSession morningSessionAt13 = new MovieSession();
         morningSessionAt13.setCinemaHall(redHall);
         morningSessionAt13.setMovie(bugs);
         morningSessionAt13.setShowTime(LocalDateTime.of(2020, 10, 10, 13, 00, 00));
         movieSessionService.add(morningSessionAt13);
+        logger.info("Morning session at 13 " + morningSessionAt13 + " was added");
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/Main.java]: you already logging it on dao layer

File: src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 3]
```diff
@@ -21,17 +24,18 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
...
             session.save(shoppingCart);
             transaction.commit();
-            return shoppingCart;
+            logger.info("Shopping cart " + shoppingCart + " was created.");
         } catch (Exception e) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't create shopping cart " + shoppingCart, e);
+            logger.error("Can't create shopping cart " + shoppingCart, e);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/cinema/dao/impl/ShoppingCartDaoImpl.java]: let's throw an exception and not replace it with loggin

**Review by dmytro-hryhoruk — COMMENTED** [review]
Apply comments to all similar places

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**